src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -2368,6 +2368,7 @@ public static ObjectMapper createMapper() {
         mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
         mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
         mapper.setDateFormat(new ISO8601DateFormat());
+        mapper.enable(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY);
```

Comment:
@aheritier I think a custom Jackson deserializer for CustomFieldValue makes more sense instead of changing the mapper used for all dtos.

Reply:
Ok Let me check how to do that :-)

Reply:
Ok it seems I could use `@JsonFormat(with = JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY)` just for the impacted field but I have to upgrade jackson .. Let me test it


https://stackoverflow.com/questions/39041496/how-to-enforce-accept-single-value-as-array-in-jacksons-deserialization-process

Reply:
If you want to avoid updating Jackson you could do something like this.. https://github.com/cloudbees/zendesk-java-client/blob/42fb254e47733b8e07a66f3fa975b890da3171f2/src/main/java/org/zendesk/client/v2/model/Action.java#L21

Reply:
@aheritier looks like the Action deserializer is doing exactly what you are trying to achieve :+1: might be nice to use the same approach to keep the models consistent.

Reply:
if we go ahead with the Jackson update I think we should also apply the same annotation fix to the Action model.